### PR TITLE
LRAC 10912

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://issues.liferay.com/browse/LRAC-10912

### CODE REVIEW

File: modules/apps/analytics/analytics-storage-service/src/main/resources/META-INF/sql/tables.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+);
+
+create table AnalyticsModelRemoveLogger (
+	mvccVersion LONG default 0 not null,
+	analyticsModelRemoveLoggerId LONG not null primary key,
+	companyId LONG,
+	userId LONG,
+	createDate DATE null,
+	modifiedDate DATE null,
+	className VARCHAR(75) null,
```
Comment (riccardo-ferrari) [inline modules/apps/analytics/analytics-storage-service/src/main/resources/META-INF/sql/tables.sql:18]: Hey @marcosapmf, I think this is not long enough, Make sure to increase it, something like:
`
	<model name="com.liferay.portal.kernel.model.ClassName">
		<field name="mvccVersion" type="long" />
		<field name="classNameId" type="long" />
		<field name="value" type="String">
			<hint name="max-length">200</hint>
		</field>
	</model>
`

File: modules/apps/analytics/analytics-storage-service/src/main/resources/META-INF/portlet-model-hints.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		<field name="body" type="Blob" />
+	</model>
+	<model name="com.liferay.analytics.storage.model.AnalyticsModelRemoveLogger">
+		<field name="mvccVersion" type="long" />
+		<field name="analyticsModelRemoveLoggerId" type="long" />
+		<field name="companyId" type="long" />
+		<field name="userId" type="long" />
+		<field name="createDate" type="Date" />
+		<field name="modifiedDate" type="Date" />
+		<field name="className" type="String" />
```
Comment (riccardo-ferrari) [inline modules/apps/analytics/analytics-storage-service/src/main/resources/META-INF/portlet-model-hints.xml:20]: Hey @marcosapmf, I think this is not long enough, Make sure to increase it, something like:
`
	<model name="com.liferay.portal.kernel.model.ClassName">
		<field name="mvccVersion" type="long" />
		<field name="classNameId" type="long" />
		<field name="value" type="String">
			<hint name="max-length">200</hint>
		</field>
	</model>
`

File: modules/apps/analytics/analytics-dxp-entity-rest-impl/rest-openapi.yaml [hunk 3]
```diff
@@ -15,6 +15,13 @@ components:
                 type:
                     type: string
             type: object
+        DXPEntityRemoveLogger:
+            properties:
```
Comment (riccardo-ferrari) [inline modules/apps/analytics/analytics-dxp-entity-rest-impl/rest-openapi.yaml:19]: It might be worth including the original removal `timestamp`

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcellustavares** [comment]
resent to https://github.com/marcosapmf/liferay-portal/pull/301

**Pull request closed without merging**